User_class.rb

```diff
@@ -0,0 +1,28 @@
+# freeze string literal: true
+
+# User view model.
+class User
+  attr_accessor :firstName, :lastName, :email
```

Comment:
you don't need these. Also, we use snake_case, not camelCase for vars.

Reply:
Old habit, will change.